### DISCUSSION TITLE
Latest Claude 3.5 version

### DISCUSSION THREAD

**Discussion by SwedenJens** [mckaywrigley/chatbot-ui#1870] [Help]
Hello,

Is there any way to use the new Claude Sonnet 3.5 version? Looking at the version in the top right corner on the "i" icon it's using the old version. I'm not super techy but I also tried running chatbot-ui on my local computer and change the version number to the latest one in the "anthropic-llm-list.ts" file but I get an error when I try to use the model.

**Comment by whatif-dev** [comment]
there are like three places to update the versions

lib/models/llm/anthropic-llm-list.ts
lib/chat-setting-limits.ts
types/llms.ts

hope that helps.

**Reply by SwedenJens** [reply]
Thanks. Updating all 3 files worked!
